usagov/vote-gov-drupal
# VOTE-1119 Make hardcoded 'Vote.gov' strings lowercase

https://cm-jira.usa.gov/browse/VOTE-1119

Insert a link to the Jira ticket (e.g VOTE-XXX).

## Description

Replace hardcoded "Vote.gov" strings to be lowercase ("vote.gov"). Any files in the old theme folder (vote_gov) and any generated files are not in scope. Any content coming from the CMS is not included.

## Deployment and testing

### Post-deploy steps

1. lando retune

### QA/Testing instructions

1. Visit the about us page and confirm all vote.gov strings (except those that are the first word in a sentence) in the social media sections of the footer.
2. Repeat for each language
3. Spot the rest of the pages and repo for any strings missed (I only found them in the social media section of the footer, but I could have missed).

## Checklist for the Developer




## Checklist for the Peer Reviewers
